src/project.ts

```diff
@@ -130,28 +131,45 @@ export class Project {
 	 * Load tiapp file
 	 *
 	 */
-	private loadTiappFile () {
+	private async loadTiappFile  () {
 		this.isTitaniumApp = false;
-		// tslint:disable-next-line:no-console
+		let error;
 		const rootPath = workspace.rootPath;
 		if (!rootPath) {
-			// tslint:disable-next-line:no-console
 			return;
 		}
 		const filePath = path.join(rootPath, TIAPP_FILENAME);
 		if (utils.fileExists(filePath)) {
+			this.isTitaniumApp = true;
```

Comment:
I think we might need to introduce another flag here like `isValid` (or similar) that we set on a valid parse of the tiapp.

If I open a project with an invalid tiapp since we now mark the project as a titaniumApp earlier, we error on startup due to trying to access data from the tiapp in [this call](https://github.com/appcelerator/vscode-appcelerator-titanium/blob/36b7708ce01e0df977216e86f983ba7f628134e7/src/extension.ts#L65)